bundle/manifests/servicemeshoperator3.clusterserviceversion.yaml

```diff
@@ -647,11 +651,15 @@ spec:
           template:
             metadata:
               annotations:
-                images.v1_24_3.cni: registry.redhat.io/openshift-service-mesh/istio-cni-rhel9:1.24.3
-                images.v1_24_3.istiod: registry.redhat.io/openshift-service-mesh/istio-pilot-rhel9:1.24.3
-                images.v1_24_3.must-gather: registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel9:3.0.1
```

Comment:
I see you replaced the must-gather image. We should document that somewhere

Reply:
I don't think this will affect users in any way. I think we can keep the must gather image referenced only from the latest version which is now 1.24.4.